### ISSUE TITLE
warning when exchange or coin is down.

### ISSUE THREAD

**Issue by cicardia** [andrefbsantos/boilr#65]
Bter is down. Maybe permanently, boiler could say when a coin was de listed, or the exchange is down right?

**Comment by dllud** [comment]
Nope. When such happens Boilr is simply unable to get data from the exchange. There is no way to distinguish the origin of that symptom. It can be due to temporary internet shortage, server congestion or many other causes.

**Comment by cicardia** [comment]
but if the coin is delisted, boilr can contact the exchange's api, it simply doesn't answer when asking for certain coins, right? how is the exchange and pair lists updated?

what i mean is, i had an alarm for a coin and exchange that are down. boilr could warn if coin or exchange is delisted.

**Comment by dllud** [comment]
Handling pair delisting is somewhat possible. Boilr gets pairs' lists when you select an exchange and its pairs haven't been updated for longer than a week (you can set the frequency on the app preferences). During that update we could build a list of delisted pairs and check if the user has alarms with any of them. We could also trigger the update when a certain alarm is able to contact the exchange but fails consistently to retrieve the last value for a given pair. 

On the other hand, handling exchange closure isn't possible. We have no way to know why the exchange isn't answering. What we're doing is to remove them on the next [libdynticker](https://github.com/andrefbsantos/libdynticker) update.

Either way I consider this bells and whistles, An active trader listens to many communication channels where such info will be available.

**Comment by cicardia** [comment]
Actualy, boilr is perfect as first warning for such issues. I agree not imperative, but a good add on.

Exchange should be greyed out if api is down, and delisted if down for too long. The same for coins. If api is on and coin is down, delist or grey out. The alarm would say, check exchange instead of prices.
